package.json

```diff
@@ -90,7 +91,8 @@
     "ember-export-application-global": "2.0.0",
     "ember-load-initializers": "2.0.0",
     "ember-maybe-import-regenerator": "0.1.6",
-    "ember-power-select": "2.3.1",
+    "ember-power-select": "2.2.3",
```

Comment:
Was the downgrade of ember-power-select intentional?

Reply:
Good catch!